src/minigames/minigame.ts

```diff
@@ -16,17 +19,31 @@ export class Minigame {
   protected cumulativeMS = 0;
   protected ticker: Ticker;
 
+  private clock: Graphics;
+  private clockArc: Graphics;
+
   constructor(
     protected readonly app: Application,
     protected readonly delegate: MinigameDelegate,
   ) {
     this.container = new Container();
+
+    this.clock = new Graphics();
+    this.clockArc = new Graphics();
+    this.clock.fillStyle = "#f0cfbb";
+    this.clock.circle(
+      CLOCK_PADDING + CLOCK_RADIUS,
+      this.app.screen.height - CLOCK_PADDING - CLOCK_RADIUS,
+      CLOCK_RADIUS,
+    );
+    this.clock.fill();
   }
 
   async attach() {
     this.ticker = new Ticker();
     if (this.lifetime !== undefined) {
       this.ticker.add(() => void this.onTick());
+      this.app.stage.addChild(this.clock);
```

Comment:
nit: should we put this on `this.container`?

Reply:
Prob but ceebs, this works.